yt_dlp/extractor/sejmpl.py

```diff
@@ -0,0 +1,124 @@
+        term, video_id = self._match_valid_url(url).group('term', 'id')
+        frame = self._download_webpage(
+            f'https://sejm-embed.redcdn.pl/Sejm{term}.nsf/VideoFrame.xsp/{video_id}',
+            video_id, headers={
+                'Referer': f'https://www.sejm.gov.pl/Sejm{term}.nsf/transmisje_arch.xsp',
+            })
+        # despite it says "transmisje_arch", it works for live streams too!
+        data = self._download_json(
+            f'https://www.sejm.gov.pl/Sejm{term}.nsf/transmisje_arch.xsp/json/{video_id}',
+            video_id, headers={
+                'Referer': f'https://www.sejm.gov.pl/Sejm{term}.nsf/transmisje_arch.xsp',
+            })
+        params = data['params']
+
+        def rfc3339_to_atende(date):
+            date = datetime.datetime.fromisoformat(date)
+            # atende uses timestamp but since 2001 instead of 1970
+            date = date.replace(year=date.year - 31)
+            # also it's in milliseconds
+            return int(date.timestamp() * 1000)
+
+        if data.get('status') == 'VIDEO_ENDED':
+            live_status = 'was_live'
+        elif data.get('status') == 'VIDEO_PLAYING':
+            live_status = 'is_live'
+        else:
+            live_status = None
+            self.report_warning(f'unknown status: {data.get("status")}')
+
+        start_time = rfc3339_to_atende(params['start'])
+        # current streams have a stop time of *expected* end of session, but actual times
+        # can change during the transmission. setting a stop_time would artificially
+        # end the stream at that time, while the session actually keeps going.
+        if live_status == 'was_live':
+            stop_time = rfc3339_to_atende(params['stop'])
+        else:
+            stop_time = None
+
+        duration = (stop_time - start_time) // 1000 if stop_time else None
+
+        entries = []
+
+        def add_entry(file):
+            if not file:
+                return
+            file = f'https:{file}?startTime={start_time}'
+            if stop_time is not None:
+                file += f'&stopTime={stop_time}'
+            stream_id = self._search_regex(r'/o2/sejm/([^/]+)/[^./]+\.livx', file, 'stream id')
+            entries.append({
+                '_type': 'url_transparent',
+                'url': file,
+                'ie_key': RedCDNLivxIE.ie_key(),
+                'id': stream_id,
+                'title': stream_id,
+                'duration': duration,
+            })
+
+        cameras = self._parse_json(
+            self._search_regex(r'(?s)var cameras = (\[.+?\]);', frame, 'camera list'),
+            video_id, js_to_json)
+        for camera in cameras:
+            add_entry(traverse_obj(camera, ('file', 'flv')))
+
+        if params.get('mig'):
+            add_entry(self._search_regex(r"var sliUrl\s*=\s*'([^']+)'", frame, 'sign language interpreter url', fatal=False))
+
+        return {
+            '_type': 'multi_video',
```

Comment:
I don't think `multi_video` is the correct type as that would cause the videos to be concatenated by default.